src/LMTR_alg.jl

```diff
@@ -115,11 +115,11 @@ function LMTR(
 
   if verbose > 0
     #! format: off
-    @info @sprintf "%6s %8s %8s %8s %7s %7s %8s %7s %7s %7s %7s %1s" "outer" "inner" "f(x)" "h(x)" "√ξ1" "√ξ" "ρ" "Δ" "‖x‖" "‖s‖" "1/ν" "TR"
+    @info @sprintf "%6s %8s %8s %8s %7s %7s %8s %7s %7s %7s %7s %1s" "outer" "inner" "f(x)" "h(x)" "√(ξ1/ν)" "√(ξ/ν)" "ρ" "Δ" "‖x‖" "‖s‖" "1/ν" "TR"
```

Comment:
Compute $\xi_1 / \nu$ once and for all instead of recomputing it many times.

The other solvers don't compute or output $\sqrt{\xi / \nu}$; only $\sqrt{\xi}$. We could think in another PR if what we really want is $\sqrt{\xi / \nu}$.

Reply:
I was stuck on this when I was rebasing on Friday. It doesn't make sense to me to only do $\sqrt{\xi}$. I believe i had changed them all in this but I guess we could revert.